### ISSUE TITLE
3.30.3 released. Sorry. I expected a PR to have been made

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/conda-smithy#1828]
### Comment:

I somewhat expected a PR to have been made when I used the rever command.

Sorry for the release without the "contact".

Would it make sense to update the instructions to:

To make a new release

1. Open a new issue with the proposed released number.
2. Wait 3-10 days for any objections from conda-forge/core
3. Then, as a member of conda-forge/core, use the `rever` command to create the new release.
   * Clone the repository locally and checkout the main branch.
   * Install the `rever` command through `mamba install rever`
   * Use `rever VERSION_NUMBER` to create a new release.
     * The release notes and tag will automatically be pushed to the main branch

**Comment by jakirkham** [comment]
That's ok. We had talked about having a release in the core chat because a few issues cropped up recently

Usually the process has been ask in the core chat if it is alright to do a conda-smithy release. If there are no objections, make the release

Think the PR feature requires setting a GitHub token locally

**Comment by hmaarrfk** [comment]
how can i find the link to the core chat.

**Comment by h-vetinari** [comment]
> how can i find the link to the core chat.

Sent you the link in a message on element.

**Comment by hmaarrfk** [comment]
hmm thanks:

MatrixError: [403] You are not invited to this room. (https://matrix-client.matrix.org/_matrix/client/v3/join/%23conda-forge_core%3Agitter.im)


my element username seems to be:

@hmaarrfk:matrix.org

should i maybe create a new account? this whole element/matrix chat transition was kinda painful.

**Comment by hmaarrfk** [comment]
on the note of 3.30.3 it seems that there is still an ongoing fix 
https://github.com/conda-forge/conda-smithy/pull/1822/

**Comment by jakirkham** [comment]
There is, but there is still discussion happening there. We can always do another release once that (and any other) fixes land

**Comment by jakirkham** [comment]
Closing as complete. Thanks for kicking this off Mark! 🙏
